src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx

```diff
@@ -358,6 +381,20 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
         >
           Annuleer
         </StyledButton>
+        {modalEmailPreviewIsOpen && (
+          <Modal
+            data-testid="emailPreviewModal"
+            open
+            onClose={closeModal}
+            title="Email Preview"
+          >
+            <EmailPreview
+              emailBody={emailBody}
+              onClose={onClose}
+              onUpdate={onUpdate}
```

Comment:
Why do both the modal and the email preview require a close callback? Shouldn't one be enough?

Reply:
because there is a cancel button in the email preview component that triggers onclose